### ISSUE TITLE
`.scss` files are incorrectly categorized as "Sass" rather than "SCSS"

### ISSUE THREAD

**Issue by LoganDark** [XAMPPRocky/tokei#913]
Looks like this project has the same mistake that [`cloc` had](https://github.com/AlDanial/cloc/issues/613) that was reported by me around 9 months back. `.scss` files are counted as "Sass" rather than "SCSS", even though SCSS and Sass are completely different.

**Comment by adam-tokarski** [comment]
> 💡 Fun fact:
Sass has two syntaxes! The SCSS syntax (.scss) is used most commonly. It's a superset of CSS, which means all valid CSS is also valid SCSS. The indented syntax (.sass) is more unusual: it uses indentation rather than curly braces to nest statements, and newlines instead of semicolons to separate them.

Taken from https://sass-lang.com/guide. 

Does these two differ enough to have it covered as separate languages?

**Comment by LoganDark** [comment]
> Does these two differ enough to have it covered as separate languages?

Yes, they are indeed two separate languages. Sass is the name of the project - SCSS is its own language, made by Sass.
